resources/views/layout/welcome.blade.php

```diff
@@ -11,6 +11,49 @@
 
  <!-- jQuery library -->
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+ 
+ <style>
+        body {
+            background-color: #7AE0BE;
+            font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif
```

Comment:
Please move the styles to /resources/sass/app.scss. You can use `npm run dev` to compile the changes.